tests/acceptance/features/webUITrashbin/trashbinDelete.feature

```diff
@@ -33,12 +33,24 @@ Feature: files and folders can be deleted from the trashbin
     # And file "lorem-big.txt" should not be listed in the trashbin folder "simple-folder" on the webUI
     But file "lorem-big.txt" should be listed on the webUI
 
+  @skipOnOCIS
+  @issue-product-188
   Scenario: Delete folders and check that they are gone
     When the user deletes folder "simple-folder" using the webUI
     And the user deletes folder "Folder,With,Comma" using the webUI
     Then folder "simple-folder" should not be listed on the webUI
     And folder "Folder,With,Comma" should not be listed on the webUI
 
+  @skipOnOC10
+  @issue-product-188
+  # after the issue is fixed delete this scenario and use the one above
+  Scenario: Delete folders and check that they are gone
+    When the user deletes folder "simple-folder" using the webUI
+    Then the error message with header "Deletion of simple-folder failed" should be displayed on the webUI
+    When the user deletes folder "Folder,With,Comma" using the webUI
+    Then folder "simple-folder" should be listed on the webUI
```

Comment:
maybe swap that around to make it more logical

Reply:
done